addon/components/code-inline.js

```diff
@@ -1,3 +1,5 @@
+/* global Prism */
```

Comment:
I'm not sure this is the most effective comment here. Is this saying that:

>Prism is already available as a global variable, so we don't need to import it explicitly here.

?